spec/classes/mod/security_spec.rb

```diff
@@ -158,25 +158,41 @@
                 dos_block_timeout: 300,
               }
             end
```

Comment:
Looks like the RuboCop annotations made it in, now that puppetlabs_spec_helper is released.

Reply:
fixed